rfcs/20200612-stream-executor-c-api.md

```diff
@@ -88,40 +88,62 @@ plan to wrap only a subset of key StreamExecutorInterface functionality.
 See proposed C API below:
 
 ```cpp
+#include <stddef.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 typedef SE_Stream_st* SE_Stream;
 typedef SE_Event_st* SE_Event;
 typedef SE_Timer_st* SE_Timer;
 typedef TF_Status* (*TF_StatusCallbackFn)(void*);
 
+#ifndef TF_BOOL_DEFINED
+#define TF_BOOL int8_t
+#endif // TF_BOOL_DEFINED
+
+#ifndef TF_OFFSET_OF_END
+#define TF_OFFSET_OF_END(TYPE, MEMBER) (offsetof(TYPE, MEMBER) + sizeof(((TYPE *)0)->MEMBER))
+#endif // TF_OFFSET_OF_END
+
 typedef struct SE_PlatformId {
+ size_t struct_size;
  void* id;  // aka stream_executor::Platform::Id
 } SE_PlatformId;
 
-typedef struct SE_TimerFns {
- const size_t struct_size;
+#define SE_PLATFORMID_STRUCT_SIZE TF_OFFSET_OF_END(SE_PlatformId, id)
+
+typedef struct SE_Timer {
```

Comment:
`SE_Timer` is currently defined by user of the API and should store pointer to timer object.
`SE_TimerFns` contains functions that take `SE_Timer` as parameter, so it can't be called `SE_Timer`. The typo was after closing bracket of the struct.

Reply:
Done!